Rest/Field/Value.php

```diff
@@ -38,28 +35,31 @@ class Value
     /** @var \Psr\Log\LoggerInterface */
     protected $logger;
 
+    /** @var array */
+    protected $fieldIdentifiers;
+
     /**
      * @param \eZ\Publish\API\Repository\ContentService $contentService
      * @param \eZ\Publish\API\Repository\ContentTypeService $contentTypeService
      * @param \EzSystems\RecommendationBundle\Rest\Field\TypeValue $typeValue
-     * @param array $parameters
      * @param \EzSystems\RecommendationBundle\Rest\Field\RelationMapper $relationMapper
      * @param \Psr\Log\LoggerInterface $logger
+     * @param array $fieldIdentifiers
      */
     public function __construct(
         ContentService $contentService,
         ContentTypeService $contentTypeService,
         TypeValue $typeValue,
-        array $parameters,
         RelationMapper $relationMapper,
-        LoggerInterface $logger
+        LoggerInterface $logger,
+        array $fieldIdentifiers
     ) {
         $this->contentService = $contentService;
         $this->contentTypeService = $contentTypeService;
         $this->typeValue = $typeValue;
-        $this->parameters = $parameters;
```

Comment:
hmm, why are you replacing parameters ? Unless you're sure we will never have to add new parameters to this service, an array is good. Fordward compatible and all.